src/index.html

```diff
@@ -16,17 +16,25 @@
       href="style.css"
     />
   </head>
-  <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+  <body class="page_body">
+    <div class="search-bar">
+      <div class="search-bar__big search-bar__field"
+           data-qa="big">
```

Comment:
The `<div>` element is not properly closed. The closing tag should be `</div>` instead of a self-closing tag. This could cause rendering issues in the browser.